test/browser/features/web_worker.feature

```diff
@@ -21,3 +21,9 @@ Scenario: setting collectUserIp option to false
   Then the error is a valid browser payload for the error reporting API
   And the event "request.clientIp" equals "[REDACTED]"
   And the event "user.id" equals "[REDACTED]"
+
+Scenario: setting autoTrackSessions option to true
```

Comment:
Similar to the above — maybe worth adding `And I should receive no sessions` to the other scenarios?